ui/src/app/cron-workflows/cron-workflow-list.tsx

```diff
@@ -222,9 +222,10 @@ export function CronWorkflowList({match, location, history}: RouteComponentProps
     );
 }
 
-function getCronNextScheduledTime(spec: CronWorkflowSpec): Date {
-    if (spec.schedule != '') {
-        return getNextScheduledTime(spec.schedule, spec.timezone);
+function getSpecNextScheduledTime(spec: CronWorkflowSpec): Date {
+    if (spec.schedule && spec.schedule.trim() !== '') {
+        const nextTime = getNextScheduledTime(spec.schedule, spec.timezone);
+        return nextTime;
```

Comment:
```suggestion
        return getNextScheduledTime(spec.schedule, spec.timezone);
```

unnecessary variable

Reply:
I'm also not sure we should have the `trim()` check here and nowhere else. I don't even think the UI should handle that tbh, and I'm not sure that would be valid YAML either

Reply:
sorry, I found an issue with next run time when using multiple schedules and I changed the code to debug. I forgot to revert. Should be good now